include/simde/chemical_system/chemical_system.hpp

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright 2022 NWChemEx-Project
+ * Copyright 2025 NWChemEx-Project
```

Comment:
Did you manually change this? If so, that's fine, but it's not necessary. I recently learned you're not actually supposed to update these things.

Reply:
Good to know. I think there are a couple of places I changed it, so I'll revert them.